### ISSUE TITLE
Stylesheet: FreeCAD Light - Selection background color slightly too dark

### ISSUE THREAD

**Issue by maxwxyz** [FreeCAD/FreeCAD#15620]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

IMO there is not enough contrast between the dark blue background color and the black text color of selected and preselected items in FreeCAD Light. 
This is visible in the menu, preferences, input fields,...
<img width="959" alt="Bildschirmfoto 2024-07-25 um 19 08 48" src="https://github.com/user-attachments/assets/dfce1af2-02b0-444d-afda-6905132b52a9">
<img width="392" alt="Bildschirmfoto 2024-07-25 um 19 10 45" src="https://github.com/user-attachments/assets/8a64796a-123b-47ac-8377-a6d46f50ccf5">
<img width="262" alt="Bildschirmfoto 2024-07-25 um 19 11 09" src="https://github.com/user-attachments/assets/6c872475-1df9-482a-8485-2245ccf50db0">
<img width="585" alt="Bildschirmfoto 2024-07-25 um 19 11 28" src="https://github.com/user-attachments/assets/913e14e9-8414-4833-9502-41b2f4551856">

### Full version info

```shell
OS: macOS 13.6.7
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38314 (Git)
Build type: Release
Branch: main
Hash: 3fa339e9eb71d34542e463b3f4581857a9e696fb
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)
Installed mods: 
  * freecad.gears 1.2.0
  * OpenTheme 2024.7.3
  * CfdOF 1.27.4
  * Defeaturing 1.2.2
  * Manipulator 1.5.7
  * fasteners 0.5.24
  * CurvedShapes 1.0.9
  * sheetmetal 0.4.22
  * Curves 0.6.41
```

### Subproject(s) affected?

Core

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MisterMakerNL** [comment]
These are user assigned colors. 
I can set them in the theme, but this would overwrite any user settings.
![image](https://github.com/user-attachments/assets/939f8b53-6e93-4949-8ecb-94efd0d0fb86)

**Comment by maxwxyz** [comment]
I thought they could be changed with a theme, like also the background color. They usually are also defined by the user but are overwritten.

**Comment by MisterMakerNL** [comment]
That is a valid point and you probably also have to change them since your colors set in light will probably not work in dark.
Do we have aby suggestion which colors?
Freecad logo colors? @FreeCAD/design-working-group ?

**Comment by FEA-eng** [comment]
This got fixed by #15586, I tested it in the newest Windows weekly.
